academy/0.0-B9lab-Blockchains/4_consensus.md

```diff
@@ -0,0 +1,112 @@
+
+![DPoS](images/00_07_delegated_pos_dark.png)
+
+Witnesses are paid for their services: They are paid fees for creating and validating blocks. This economic incentive to become a witness also leads to competition potentially increasing with each new member because the number of witnesses is limited.
+
+In case a witness misbehaves, the network's community is able to withdraw their votes for a single witness, i.e. fire the witness. Witnesses that no longer hold enough votes lose their income basis.
+
+Alongside ascribing the role of witnesses to some participants, DPoS networks also elect **"delegates"**. Delegates are a group of participants that supervise network governance and performance, and propose changes that are then voted on by the entire network.
+
+Many consider DPoS algorithms superior to PoW and PoS because of their fast block creation, a high degree of security, energy efficiency, level of integrity, and democratic structure. However, DPoS systems are less decentralized than PoW and PoS systems because they have fixed validator sets and higher barriers to entry.
+
+## Practical Byzantine Fault Tolerance (PBFT)
+
+PBFT was published in 1999 and arose from academia. It is a three-phase protocol, in which the client sends a request to a so-called primary. 
+
+In the first phase, the primary broadcasts the request with a sequence number to the replicas. Then the replicas agree on the sequence number and create a message. 
+
+When a threshold of agreement is reached by the replicas, which is akin to sufficient co-signers, the message is verified. The replicas agree on the overall order of transactions within a view. The broader network is informed about transaction blocks when they are finalized. That is to say, signed by sufficient replicas to be sure that the block is finalized. 
+
+This of course is a very simplified presentation.
+
+<HighlightBox type="reading">
+
+    You can access the paper by Miguel Castro and Barabara Liskov [here](http://pmg.csail.mit.edu/papers/osdi99.pdf). There is also an intuitive [presentation](http://www.comp.nus.edu.sg/~rahul/allfiles/cs6234-16-pbft.pdf).
+
+</HighlightBox>
+
+## Tendermint ##
+
+While most consensus implementations are tightly coupled with a particular blockchain project, Tendermint is a toolkit that focuses on simplifying the process of creating blockchains with different properties and it includes Tendermint consensus. As the reader might guess, Cosmos relies on Tendermint consensus which offers the most mature Byzantine Fault Tolerance, transaction finality as well as a great deal of flexibility - flexibility that is passed through to the creators of custom blockchains built with the Cosmos SDK. 
+
+[Tendermint](./1-what-is-cosmos/blockchain-and-cosmos.md) will be explored in detail later on. 
+
+## Many others ##
```

Comment:
I moved this part in my commits to the beginning of the section to give it a seamless flow overall.